src/Accelerators/Accelerator.cpp

```diff
@@ -25,5 +25,25 @@ const llvm::SmallVectorImpl<Accelerator *> &Accelerator::getAccelerators() {
   return acceleratorTargets;
 }
 
+// Help to print accelerator kinds.
+static std::map<Accelerator::Kind, std::string> mapKind2Strings;
+
+std::ostream &operator<<(std::ostream &out, const Accelerator::Kind kind) {
+  if (mapKind2Strings.size() == 0) {
```

Comment:
[Suggestion]:  `if (mapKind2Strings.empty()) {`

Reply:
yes, like it better